src/app/product-detail/product-detail.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="card mb-3 top-100 mx-auto" *ngIf="selectedProduct" style="max-width: 640px;">
+    <div class="row g-0">
+      <div class="col-md-4">
+        <img [src]="selectedProduct.icon" class="img-fluid rounded-start" alt="icon">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body">
+          <h5 class="card-title">{{selectedProduct.title}}</h5>
+          <p class="card-text">{{selectedProduct.teaser}}</p>
+<!--           <p class="card-text"><small class="text-muted">Last updated 3 mins ago</small></p>
+ -->          <a href="#" class="btn btn-primary" [routerLink]="['/configurator', selectedProduct.id]">Configurate</a>
```

Comment:
Also please don't push commented out html or code

Reply:
I got it ))